backend/core/src/migration/1640879376543-favoriteID2.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
```

Comment:
Do we need both migration files here?

Reply:
Yes the first migration reverses the earlier ManyToMany favoriteIDs, and the second correctly updates the JSONB array.  I tried removing them both and just creating one new one, but it failed to recognize the changes to the backend.